docs/BR.md

```diff
@@ -1008,6 +1056,62 @@ CAs are permitted to treat a record lookup failure as permission to issue if:
 
 CAs MUST document potential issuances that were prevented by a CAA record in sufficient detail to provide feedback to the CAB Forum on the circumstances, and SHOULD dispatch reports of such issuance requests to the contact(s) stipulated in the CAA iodef record(s), if present. CAs are not expected to support URL schemes in the iodef record other than mailto: or https:.
 
+#### 3.2.2.9 Multi-Perspective Issuance Corroboration
+
+Multi-Perspective Issuance Corroboration attempts to corroborate the determinations made by the Primary Network Perspective from multiple additional Network Perspectives before subscriber certificate issuance. This process can improve protection against equally-specific prefix Border Gateway Protocol (BGP) attacks or hijacks.
+
+Each corroborating Network Perspective MUST independently verify: 
```

Comment:
I have a question about the use of the word "verify" in this statement.  To me, that implies that every remote perspective must check the value found there is what is expected, is this intentional?

Why I'm asking is that this language seems to imply that when doing MPIC for DNS changes, the CA must say to the remote perspectives "Go and check the DNS TXT record for example.com.  The value you must see there is <random_value>.  Return true/false depending on if you saw <random_value> or not."  There the remote perspective has independently verified the DNS change.

However, this only really works like this for agreed upon changes to websites/DNS changes.  For all the contact information validation methods the call _must_ be "Go and get the email contact information for example.com from the domain name registrar" because there's not anything to verify as the goal is to make sure that the contact information is the same from many perspectives.

For the sake of simplicity, and cleanliness of interfaces, it would be nice to retain the option to have every remote network perspective call to be "Go and get <thing> from <record_type> for example.com".  The next step would be having the CA look at the responses from all perspectives and then determining if the primary perspective and x of y remote perspectives all saw the correct value and then any different behaviour for contact information vs expected changes could be applied.  However, I'm not sure this approach is valid if "Each corroborating Network Perspective MUST independently _verify_ [ed. my emphasis]" the expected random value/token.

Reply:
Pushed new language in https://github.com/ryancdickson/staging/commit/ba979635b3b31fe842b226cb2a66bc16ede78036.

Marking conversation as resolved, thanks for your comment @robplee.